### DISCUSSION TITLE
New firmware release (v1.12.0) available on GitHub

### DISCUSSION THREAD

**Discussion by dmcnaugh** [thehighnibble/thehighnibble.github.io#74] [Announcements]
# [June 2023 Release](https://github.com/thehighnibble/firmware/releases/tag/v1.12.0)

This is a **minor** release that:

- Focuses on stability and performance improvements
- Fixes bugs
- Enables disk images to be renamed in the `LIB:` window on the Desktop UI 

See the release notes for more details.

**Comment by PaulKlasmann** [comment]
I downloaded the firmware folder but I see only 3 .md files, and no .bin for the upddate.

**Reply by nbreeden2** [reply]
Did you download FIRMWARE.zip from the bottom of this page?
https://github.com/thehighnibble/firmware/releases/tag/v1.12.0
I just downloaded and unzipped it and see these files:
bootloader.bin
cromemcosim_esp32.bin
flash.sh
FLASHING.md
imsaisim_esp32.bin
ota_data_initial.bin
partition-table.bin

**Reply by PaulKlasmann** [reply]
Thank you, that worked.

**Reply by nbreeden2** [reply]
Awesome, glad it worked.
